public/javascripts/SVValidate/src/Main.js

```diff
@@ -179,7 +179,12 @@ function Main (param) {
             svv.gsvOverlay = new GSVOverlay();
             svv.keyboard = new Keyboard(svv.ui.validation);
             svv.labelVisibilityControl = new LabelVisibilityControl();
-            svv.speedLimit = new SpeedLimit(svv.panorama.getPanorama(), svv.panorama.getPosition, () => false, svv.panoramaContainer);
+            const labelTypeTemp = param.labelList[0].getAuditProperty('labelType');
+            // Checking to see if the labeltype is NoCurbRamp before adding speedlimit.
+            if (labelTypeTemp == 'NoCurbRamp') {
```

Comment:
This is also being checked only once when loading the page. But the label type can change in subsequent missions. While testing, the first mission I had was for Missing Curb Ramps, so speed limits correctly showed up. But my next mission was for Crosswalks and I was still seeing the speed limits.